src/esp/gfx_batch/DepthUnprojection.h

```diff
@@ -151,6 +151,21 @@ consumers expect zeros for things that are too far.
 void unprojectDepth(const Magnum::Vector2& unprojection,
                     Corrade::Containers::ArrayView<Magnum::Float> depth);
 
+/**
+@brief Unproject depth values
+@param[in] unprojection Unprojection coefficients from
+    @ref calculateDepthUnprojection()
+@param[in,out] depth    Depth values in range @f$ [ 0 ; 1 ] @f$
+
+See @ref calculateDepthUnprojection() for the full algorithm explanation.
+Additionally to applying that calculation, if the input depth is at the far
+plane (of value @cpp 1.0f @ce), it's set to @cpp 0.0f @ce on output as
+consumers expect zeros for things that are too far.
+*/
+void unprojectDepth(
+    const Magnum::Vector2& unprojection,
+    const Corrade::Containers::StridedArrayView2D<Magnum::Float>& depth);
```

Comment:
No, nono, I didn't mean an overload. Because this way you end up with a copy of the same code that has _no test coverage whatsoever_. That's bad for several reasons.

Delete the original, keep just this one. The `ArrayView` is implicitly convertible to a `StridedArrayView`, so existing call sites should hopefully just work. If not, do it [as per my suggestion in the previous comment](https://github.com/facebookresearch/habitat-sim/pull/2129/commits/aec334197112dd96a3bc4b2d1a3a8e598b6cc024#r1235492571), i.e. replace `arrayCast<Float>(image.data())` with `image.pixels<Float>()`.